src/App.js

```diff
@@ -1,79 +1,15 @@
 import React from 'react';
 import './App.css';
+import Welcome from './components/welcome/welcome';
+import Header from './components/header/header';
+import Article from './components/article/article';
```

Comment:
файл компонента должен называться с большой буквы
```suggestion
import Welcome from './components/welcome/Welcome';
import Header from './components/header/Header';
import Article from './components/article/Article';
```